File: src/main/java/com/minone/bogus/config/Configuration.java
package com.minone.bogus.config;

import com.minone.bogus.web.SubnetFilter;
import org.springframework.boot.SpringBootConfiguration;
import org.springframework.boot.web.servlet.FilterRegistrationBean;
import org.springframework.context.annotation.Bean;

@SpringBootConfiguration
public class Configuration {

    @Bean
    public FilterRegistrationBean<SubnetFilter> subnetFilter() {

        FilterRegistrationBean<SubnetFilter> registration = new FilterRegistrationBean<>();

        registration.setFilter(new SubnetFilter());
        registration.addUrlPatterns("/health");

        return registration;
    }
}

File: src/main/java/com/minone/bogus/config/Configuration.kt
package com.minone.bogus.config

import com.minone.bogus.web.SubnetFilter
import org.springframework.boot.SpringBootConfiguration
import org.springframework.boot.web.servlet.FilterRegistrationBean
import org.springframework.context.annotation.Bean

@SpringBootConfiguration
class Configuration {
    @Bean
    fun subnetFilter(): FilterRegistrationBean<SubnetFilter> =
        FilterRegistrationBean<SubnetFilter>().apply {
            this.filter = SubnetFilter()
            this.addUrlPatterns("/health")
        }
}